examples/liquid.ini

```diff
@@ -11,6 +11,11 @@ consul_socket = ../cluster/var/consul.socket
 # Domain for the liquid bundle.
 domain = liquid.example.org
 
+# User-friendly title.
+# Defaults to the domain with no spaces, capitalized.
```

Comment:
How about we add more magic and remove the last bit of the domain name? That would actually yield more relevant vaues.

Reply:
overkill